docs/app-modules/interchain-accounts/overview.md

```diff
@@ -19,14 +19,14 @@ Developers looking to build upon Interchain Accounts must write custom logic in
 
 Regular accounts use a private key to sign transactions on-chain. Interchain Accounts are instead controlled programmatically by separate chains via IBC transactions. Interchain Accounts are implemented as sub-accounts of the interchain accounts module account. 
 
-# Concepts 
+## Definitions 
```

Comment:
Aligns with same pattern used in `docs/ibc/middleware/develop.md`

Reply:
I think we should keep it as `Concepts`

Reply:
I have no preference

Reply:
I don't have a problem naming the heading as "Concepts" but I do think we should be consistent throughout the docs. So that would mean renaming "Definitions" heading used within the middleware docs to "Concepts". These kind of sections should, in my opinion, act as a sort of glossary of words/terminology to give context on the language used for a particular topic. Either heading is fine but we should be consistent.

Reply:
I think we use `Concepts` in other docs and in some of the SDK docs also.